### ISSUE TITLE
post grib2 files to opendap

### ISSUE THREAD

**Issue by jasonfleming** [StormSurgeLive/asgs#665]
For convenience in performing retrospective analyses, including both nam-only and blended winds, we need to post grib2 files for opendap service. This will not require a lot of storage on the opendap servers, because we now do partial grib downloading, so the files are only approximately 500kB each.

**Comment by wwlwpd** [comment]
this would be considered part of the set of _input_ files for each model run?

**Comment by jasonfleming** [comment]
Yes, the raw input, rather than something that can be read directly into ADCIRC. The main reason is so that the data can be re-used for other types of runs. It is analogous to us keeping our own archive of BEST track files and NHC forecast/advisories.

**Comment by jasonfleming** [comment]
I took out the triage/grooming tag b/c this is a lowerish nice-to-have.
